### PR TITLE
Add tests for state list drawer

### PR DESCRIPTION
**Pull request by upalatucci**
Add a cypress test to ensure that the nns table works, the interface is clickable and the Drawer opens showing good information.

### CODE REVIEW

**Review by avivtur — COMMENTED** [review]
Hi @upalatucci 
I'd suggest adding a config to have a similar prop like we have in the kubevirt-plugin `data-test-id` and assigning the components you want to get with unique id's, that should improve readability on this test and will make our future lives easier :)

**Comment by upalatucci** [comment]
thanks @avivtur fixed

File: cypress/e2e/StatusList.spec.cy.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const EXPAND_INTERFACES_LIST_TEST_ID = '#expand-interfaces-list0';
```
Comment (pcbailey) [inline cypress/e2e/StatusList.spec.cy.ts]: It would be cleaner to keep the constants in a separate file.

Comment (upalatucci) [reply cypress/e2e/StatusList.spec.cy.ts]: fixed

**Pull request merged**